Sources/SwiftSyntax/Syntax.swift

```diff
@@ -225,7 +225,7 @@ extension SyntaxProtocol {
 
   /// Return this subtree with this node as the root, ie. detach this node
   /// from its parent.
-  public func detach() -> Self {
+  public var detached: Self {
```

Comment:
I think we should keep the the function, but mark it as deprecated. 
Then next to this we add the new var.

Otherwise it might break for people using this method

Reply:
Thank you for your review! I made the function deprecated.